sass/pages/check-mail/_check_mail.scss

```diff
@@ -0,0 +1,56 @@
+.check-body{
+	background-color: #F7F7F7;
+
+	.check-section{
+		background-color: $white;
+		text-align: center;
+
+		.check-div{
+			width: 30%;
+			@include center(true,true);
+			background-color: $white;
+
+			.check-form{
+				
+				.check-field{
+					border-radius: 7.5px;
+					border: none;
+					-webkit-box-shadow: 0px 0px 10px 2px rgba(184,175,184,0.51);
+					-moz-box-shadow: 0px 0px 10px 2px rgba(184,175,184,0.51);
+					box-shadow: 0px 0px 10px 2px rgba(184,175,184,0.51);
+
+					.check-h1{
+						height: 5.14%;
+						color: #757779;
+						font-family: Muli-Light;
+						font-size: 20px;
+						font-weight: 300;
+						line-height: 27px;
+						margin-bottom: 50px;
+						margin-top: 50px;
+					}
+					
+					.check-p{
+						height: 5.14%;
+						color: #757779;
+						font-family: Muli-Light;
+						font-size: 15px;
+						font-weight: 300;
+						margin-bottom: 50px;
```

Comment:
margin: 50px 0;    would have the same effect as including margin-top and margin-bottom separately. Read about properties syntax.
You could set the four values like:
- margin/padding: top  right  down  left;

In that order.

Or set just two and it will read as:
- margin/padding: top&bottom  left&right;

Even if you write three values will work if you need:
- margin/padding:  top  left&right  bottom;